src/Requests/Uptime/GetUptimeRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace OhDear\PhpSdk\Requests\Uptime;
+
+use OhDear\PhpSdk\Dto\Uptime;
+use OhDear\PhpSdk\Helpers\Helpers;
+use Saloon\Enums\Method;
+use Saloon\Http\Request;
+use Saloon\Http\Response;
+
+class GetUptimeRequest extends Request
+{
+    protected Method $method = Method::GET;
+
+    public function __construct(
+        protected int $monitorId,
+        protected string $startedAt,
+        protected string $endedAt,
```

Comment:
Should this support choosing the split type? 👀

Reply:
Feel free to add it 👍

Reply:
Realised this isn't documented, so I'll exclude it from the SDK too.